chart/kubeapps/templates/ingress.yaml

```diff
@@ -1,38 +1,36 @@
 {{- if .Values.ingress.enabled -}}
-{{- $fullName := include "kubeapps.fullname" . -}}
-{{- $ingressPath := .Values.ingress.path -}}
+{{- range .Values.ingress.hosts }}
```

Comment:
so this is creating an ingress per host? Why don't we want to use the multi-host capabilities of a single ingress?

Having different ingresses could mean that cert-manager instead of issuing a cert with multiple CN will potentially do different certs no?

Reply:
Yes, I think it's better to use `multi-host` since it requires issuing many certificates